src/views/ServerStartView.vue

```diff
@@ -2,21 +2,50 @@
   <div
     class="font-sans flex flex-col justify-center items-center h-screen m-0 text-neutral-300 bg-neutral-900 dark-theme pointer-events-auto"
   >
-    <h2 class="text-2xl font-bold">{{ ProgressMessages[status] }}</h2>
+    <div
+      v-if="status == ProgressStatus.ERROR"
+      class="flex flex-col items-center my-[60px]"
+    >
+      <h1 class="text-[60px] my-0">{{ $t('serverStartView.errorTitle') }}</h1>
```

Comment:
We mostly uses REM over PX in this project. Unless you have good reason to use px, please stick to rem.

> rem stands for “root em“ and refers to the font size of the document’s root element. This means that the size of an element defined in rem will be relative to the size of the root font.

> px, on the other hand, stands for “pixels” and refers to the fixed size of an element in terms of the number of pixels on a screen. This means an element defined in px will always have the same size, regardless of the root font size.

> So why would you use one over the other?

> Well, using rem can be useful if you want to create flexible and responsive design layouts that adjust to the root font size. This can be helpful if you’re designing for different devices with various screen sizes.

> On the other hand, using px can be useful if you want complete control over the size of an element and ensure that it always stays the same. This can be helpful if you’re designing something like a graphic or an icon that needs to be a specific size.

> In conclusion, rem and px are two units of measurement in CSS that can be used for different purposes. rem is useful for creating a flexible and responsive design, while px is useful for having precise control over the size of an element.

Reply:
I was working off the Figma file (https://www.figma.com/design/WyLTXXwRnxhdnmqLwDJY60/Comfy's-UI?node-id=923-2901&node-type=frame&t=wmeL85m4NQ6oWJPz-0) that defines pretty much everything in this PR. I don't mind changing it as you suggested but shouldn't this go through some review process?

Reply:
Sorry we probably miscommunicated the expectation on figma design. Because we didn't purchase the figma design assets from PrimeVue, all figma mocks only mocks the user interaction, not style. So do whatever you think to make it look good while keeping the interaction logic the same.